docs/src/examples.md

```diff
@@ -281,6 +282,8 @@ f_square(v) = v^2 * 1.5 - v^2
 println("Applying function to y_q: ", sum(f_square.(y_q)))
 ```
 
+See [Home > Arrays](https://ai.damtp.cam.ac.uk/dynamicquantities/stable/#arrays) for more.
```

Comment:
This is an absolute path URL; preferably this should be relative in case I ever change the base URL

Reply:
Oh, roger, how's this? [12b20d2](https://github.com/SymbolicML/DynamicQuantities.jl/pull/178/commits/12b20d2723510988eab944295039de2b04ca9bfc)